build.gradle

```diff
@@ -1,6 +1,7 @@
 buildscript {
   ext.versions = [
-    'supportLibrary': '28.0.0-alpha3',
+    'annotation_version': '1.0.0',
```

Comment:
Can you actually just inline these now? The version isn't being re-used anymore so there's no point in keeping this separate.

Reply:
I fix it.

Reply:
Sorry, I meant can these be deleted and put directly into the dependencies in `swirl/build.gradle`. 

Ex:
```
implementation "androidx.annotation:annotation:1.0.0"
```

Reply:
Oh sorry, i didn't understand you, i have pushed the fix.